backend/src/app/migrations/sql/0073-mod-share-link-table.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE share_link
+  ADD COLUMN who_comment text NOT NULL DEFAULT('team'),
+  ADD COLUMN who_inspect text NOT NULL DEFAULT('team'),
+  DROP COLUMN flags;
```

Comment:
I'm not clearly understand this change, the previous code already contemplated a permissions system like the one implemented in this PR, it was intended to be an extensible system... But on this PR the approach is removed and replaced with the ad-hoc one.

Reply:
We have talked about this, I'm closing the comment